test/src/main/java/org/apache/accumulo/test/functional/AmpleConditionalWriterIT.java

```diff
@@ -745,58 +751,70 @@ public void testCompacted() {
 
       var ctmi = new ConditionalTabletsMutatorImpl(context);
 
+      FateInstanceType type = FateInstanceType.fromTableId(tid);
+      FateId fateId55L = FateId.from(type, 55L);
+      FateId fateId45L = FateId.from(type, 45L);
+      FateId fateId56L = FateId.from(type, 56L);
+      FateId fateId65L = FateId.from(type, 65L);
+      FateId fateId75L = FateId.from(type, 75L);
```

Comment:
Reordered the vars to be sorted.

```suggestion
      FateId fateId45L = FateId.from(type, 45L);
      FateId fateId55L = FateId.from(type, 55L);
      FateId fateId56L = FateId.from(type, 56L);
      FateId fateId65L = FateId.from(type, 65L);
      FateId fateId75L = FateId.from(type, 75L);
```